stl/inc/xutility

```diff
@@ -6674,13 +6680,15 @@ _EXPORT_STD template <class _ExPo, class _FwdIt, class _Pr, _Enable_if_execution
 _NODISCARD _FwdIt max_element(_ExPo&&, _FwdIt _First, _FwdIt _Last, _Pr _Pred) noexcept /* terminates */ {
     // find largest element
     // not parallelized at present, parallelism expected to be feasible in a future release
+    _REQUIRE_CPP17_MUTABLE_ITERATOR(_FwdIt);

```

Comment:
`max_element` and `min_element` aren't modifying, so they shouldn't require mutable iterators. Occurs 4 times total here.